src/components/Tracker.js

```diff
@@ -0,0 +1,141 @@
+import React, {useEffect, useState} from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+import axios from "axios";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheck, faTimes} from '@fortawesome/free-solid-svg-icons'
+import {searchXref} from '../actions/searchActions';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+
+const restUrl = process.env.REACT_APP_RESTAPI;
+
+const searchMissingFiles = (mod_abbreviation) => {
+  return dispatch => {
+    axios.get(restUrl + '/reference/missing_files/'+mod_abbreviation)
+        .then(res => {
+          dispatch(setMissingFileResults(res.data));
+        })
+        .catch();
+    }
+}
+
+function getOktaModAccess(oktaGroups) {
```

Comment:
This should probably just import the other module that exists, so we don't have to maintain 2 copies when things change ?  We're probably going to be changing something for okta Tester group, which would override access with whatever they set manually through the UI.